File: Linear_search/Kotlin/linear_search.kt
fun linearSearch(arr:IntArray, target: Int): Int {
    for (i in arr.indices){
        if (arr[i] == target) {
            return i
        }
    }
    return -1
}

fun main() {
    val num = intArrayOf(14, 176, 1983, 67, 666)
    val required = 666
    val res = linearSearch(num, required)

    println(if (res != -1) "The element $required was found at the $res" else "The element $required was not found")
}

File: Linear_search/PHP/linear_search.php
<?php

function linierSearch($arr, $target){
    for ($i = 0; $i < count($arr); $i++) {
        if($array[$i] === $target) {
            return $i;
        }
    }
    return -1;
}

$num = [12, 1252, 174, 21, 9];
$required = 174;

$res = linierSearch($num, $required);
if ($res != -1) {
    echo "The element {$required} was found at position {$res}\n";
} else {
    echo "The element {$required} was not found\n";
}
?>